README.md

```diff
@@ -8,6 +8,7 @@ See the [User Manual](https://prestodb.io/docs/current/) for deployment instruct
 
 * Mac OS X or Linux
 * Java 8 Update 92 or higher (8u92+), 64-bit
+  * Presto supports Oracle Java and OpenJDK
```

Comment:
`Oracle Java` -> `Oracle JDK`

We can even put this new line on the previous line (as this is the only sub-bullet we have now):

```
Java 8 Update 92 or higher (8u92+), 64-bit. Both Oracle JDK and OpenJDK are supported.
```

Reply:
done.